scripts/bash/enceladus_env.template.sh

```diff
@@ -26,7 +26,7 @@ SPARK_SUBMIT="$SPARK_HOME/bin/spark-submit"
 
 HDP_VERSION="2.7.3"
 
-SPARK_JOBS_JAR="enceladus-spark-jobs.jar"
+SPARK_JOBS_JAR="${SPARK_JOBS_JAR:="enceladus-spark-jobs.jar"}"
```

Comment:
Generally, I like the solution and it will work. but then this doesn't seem like a plain environment script.

How about introducing `SPARK_JOBS_JAR_OVERRIDE` or something that is then used in `run_standardization.sh` like `export JAR=${SPARK_JOBS_JAR_OVERRIDE:=$SPARK_JOBS_JAR}`

Basically using your solution but in a lower place.

Reply:
So then it would be modified in more places?

Reply:
I don't want evaluation and user overriding in ENV script. Yes, it would be in more places. 2 places vs 1. `run_standardization.sh` and  `run_conformance.sh`. Or you can try and do it in `run_enceladus.sh`.

Now that I am looking into `run_enceladus.sh` I see 

```shell
    --jar)
    JAR="$2"
    shift 2 # past argument and value
    ;;
```
What does it do? Does it do the same?

Reply:
Yes, so I think it's about modifying `run_standardization.sh` and `run_conformance.sh` to include this override.